### PR TITLE
implement admin jobs filtering

### PR DESCRIPTION
**Pull request by martenson**
pulled out of https://github.com/galaxyproject/galaxy/pull/15237 in an effort to simplify things

~~needs to wait for https://github.com/galaxyproject/galaxy/pull/16017~~

## How to test the changes?
(Select all options that apply)
- [x] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: client/src/components/Indices/filtersMixin.js [hunk 1]
```diff
@@ -4,15 +4,40 @@ export default {
     components: {
         IndexFilter,
     },
+    props: {
```
Comment (mvdbeek) [inline client/src/components/Indices/filtersMixin.js:7]: Could you refactor the mixin to a composable ?

File: test/integration_selenium/test_admin_jobs.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+from .framework import (
+    selenium_test,
+    SeleniumIntegrationTestCase,
+)
+
+SCRIPT_DIRECTORY = os.path.abspath(os.path.dirname(__file__))
+JOB_CONFIG = os.path.join(SCRIPT_DIRECTORY, "test_admin_jobs_job_conf.yml")
+
+
+class TestAdminDependencyContainers(SeleniumIntegrationTestCase):
```
Comment (jmchilton) [inline test/integration_selenium/test_admin_jobs.py]:
```suggestion
class TestAdminJobsGrid(SeleniumIntegrationTestCase):
```

File: client/src/components/Indices/filtersMixin.js (hunk 1 above)
Comment (mvdbeek) [reply client/src/components/Indices/filtersMixin.js:7]: We've talked about this and the mixin itself brings in a lot of other things via the IndexFilter component, refactoring without converting everything to composition API is tricky. Given the original PR started some time ago before composition was an option we should probably keep the mixin for now.

Comment (martenson) [reply client/src/components/Indices/filtersMixin.js:7]: Alright, makes it easier for this PR. Further refactoring of this mixin should be pretty separated from the rest of the work in https://github.com/galaxyproject/galaxy/pull/15237 thus independent. Thanks a lot for the discussion.

**Comment by martenson** [comment]
This is ready to go once the tests pass. The mixin refactoring is treated independently from this PR.

**Comment by dannon** [comment]
We should definitely make every effort to eliminate mixins as we modernize code, but especially given the context here, that it's being broken out of a larger project that you'll follow up on, I think this is fine.  Thanks @martenson

**Review by dannon — APPROVED** [review]

**Pull request merged**